soh/soh/OTRGlobals.cpp

```diff
@@ -128,6 +128,21 @@ Color_RGB8 kokiriColor = { 0x1E, 0x69, 0x1B };
 Color_RGB8 goronColor = { 0x64, 0x14, 0x00 };
 Color_RGB8 zoraColor = { 0x00, 0xEC, 0x64 };
 
+typedef struct {
+    /* 0x00 */ Color_RGBA8 inner;
+    /* 0x04 */ Color_RGBA8 outer;
+} NaviColor; // size = 0x8
+
+NaviColor sNaviColorList[] = {
+    { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },         { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },
+    { { 255, 255, 255, 255 }, { 0, 0, 255, 0 } },     { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },
+    { { 150, 150, 255, 255 }, { 150, 150, 255, 0 } }, { { 255, 255, 0, 255 }, { 200, 155, 0, 0 } },
+    { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },         { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },
+    { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },         { { 255, 255, 0, 255 }, { 200, 155, 0, 0 } },
+    { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },         { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },
+    { { 0, 255, 0, 255 }, { 0, 255, 0, 0 } },
+};
```

Comment:
Following on from the other review comment, you could then write out all the color options like defaultIdlePrimaryColor (or maybe defaultIdleInnerColor) and populate the array that way, e.g.

```cpp
NaviColor sNaviColorList[] = {
    { defaultInnerColor, defaultOuterColor},
     ... ,
    {defaultIdleInnerColor, defaultIdleOuterColor}, ...
}
```